spikeextractors/extractors/shybridextractors/shybridextractors.py

```diff
@@ -0,0 +1,174 @@
+import os
+
+import numpy as np
+
+from spikeextractors import RecordingExtractor
+from spikeextractors import SortingExtractor
+from spikeextractors.extractors.bindatrecordingextractor import BinDatRecordingExtractor
+from spikeextractors.extraction_tools import save_to_probe_file
+
+try:
+    import hybridizer.io as sbio
+    import hybridizer.probes as sbprb
+    HAVE_SBEX = True
+except ImportError:
+    HAVE_SBEX = False
+
+class SHYBRIDRecordingExtractor(RecordingExtractor):
+    extractor_name = 'SHYBRIDRecordingExtractor'
+    installed = HAVE_SBEX
+    extractor_gui_params = [
+        {'name': 'recording_fn', 'type': 'file', 'title': "Full path to hybrid recording (.bin)"},
+    ]
+
```

Comment:
can you add an installation_mesg?

Reply:
The installation procedure is still to be improved, let me get back to this point later.